### DISCUSSION TITLE
Next release (Django 4.1 support)

### DISCUSSION THREAD

**Discussion by MaertHaekkinen** [encode/django-rest-framework#8638] [General] [answered]
Hello!

Is a new release planned soon? I think that a release is due, since during the last 9+ months quite a bit has changed.
I am interested in Django 4.1 support (removal of pytz), which hasn't been released yet.

Thank you for your attention!

**Comment by lovelydinosaur** [comment, marked as answer]
Follow https://github.com/encode/django-rest-framework/pull/8599

**Reply by MaertHaekkinen** [reply]
Thanks!
